crates/valence/src/weather.rs

```diff
@@ -0,0 +1,338 @@
+use std::ops::Range;
+
+use bevy_ecs::prelude::*;
+use bevy_ecs::schedule::SystemConfigs;
+use valence_protocol::packet::s2c::play::game_state_change::GameEventKind;
+use valence_protocol::packet::s2c::play::GameStateChangeS2c;
+
+use crate::packet::WritePacket;
+use crate::prelude::*;
+
+pub const WEATHER_LEVEL_RANGE: Range<f32> = 0_f32..1_f32;
```

Comment:
Is not the name of this constant too wordy? Maybe just `WEATHER_LEVEL` would be better?